payload-examples/api.github.com/pull_request.assigned.payload.json

```diff
@@ -366,6 +366,7 @@
       }
     },
     "author_association": "OWNER",
+    "active_lock_reason": null,
```

Comment:
This is either `resolved`, `off-topic`, `too heated`, `spam` when it's locked and `null` when it's unlocked

Reply:
let's update that in the schema at #186 before we merge the PR

Reply:
I've done that at https://github.com/heuels/webhooks/commit/8124337883c5937c688c3d57a24f6db25651b8c6